misc/README.md

```diff
@@ -7,6 +7,9 @@ The process for updating the `make e2e`'s input is manual.  Each time a go-algor
 | Run go-algorand `e2e_subs` tests to produce new Indexer E2E inputs.<br/>* See script comments for local environment setup.<br />* Expect tests to run for ~5-10min.                                                                                                            | `bash misc/buildtestdata.sh`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
-| Locally confirm Indexer E2E tests pass with new inputs.<br />* _Warning_:  Only use these commands locally because they're _not_ secure.<br />* The commands directly invoke `e2elive.py` to increase local iteration speed. `make e2e` spins up a Docker compose environment. | * `docker run -it --rm --name some-postgres -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_USER=$USER -e POSTGRES_DB=indexer_db_e2e postgres` <br/>* `python misc/e2elive.py --source-net $E2EDATA/net_done.tar.bz2 --connection-string "host=localhost port=5432 user=$USER dbname=indexer_db_e2e sslmode=disable"`<br /> * Look for `e2elive.py` output starting with: `indexer e2etest OK`.                                                                                                                                                            |
+| Locally confirm Indexer E2E tests pass with new inputs.<br />* _Warning_:  Only use these commands locally because they're _not_ secure.<br />* The commands directly invoke `e2elive.py` to increase local iteration speed. `make e2e` spins up a Docker compose environment. <br><b>*If you are running into dependency issues please consult the help section below</b></br>| * `docker run -it --rm --name some-postgres -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_USER=$USER -e POSTGRES_DB=indexer_db_e2e postgres` <br/>* `python misc/e2elive.py --source-net $E2EDATA/net_done.tar.bz2 --connection-string "host=localhost port=5432 user=$USER dbname=indexer_db_e2e sslmode=disable"`<br /> * Look for `e2elive.py` output starting with: `indexer e2etest OK`.                                                                                                                                                            |
```

Comment:
@algoidurovic Minor, optional presentation change for consistency:  Add space before bullet point.

```suggestion
| Locally confirm Indexer E2E tests pass with new inputs.<br />* _Warning_:  Only use these commands locally because they're _not_ secure.<br />* The commands directly invoke `e2elive.py` to increase local iteration speed. `make e2e` spins up a Docker compose environment. <br><b>* If you are running into dependency issues please consult the help section below</b></br>| * `docker run -it --rm --name some-postgres -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_USER=$USER -e POSTGRES_DB=indexer_db_e2e postgres` <br/>* `python misc/e2elive.py --source-net $E2EDATA/net_done.tar.bz2 --connection-string "host=localhost port=5432 user=$USER dbname=indexer_db_e2e sslmode=disable"`<br /> * Look for `e2elive.py` output starting with: `indexer e2etest OK`.                                                                                                                                                            |
```